server/app/models/List.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+
+const options = {
+  timestamps: true,
+};
+
+const ListSchema = mongoose.Schema({
+  name: {
+    type: String,
+    required: [true, 'Please enter a list name'],
+  },
+  coverUrl: {
+    type: String,
+    required: [true, 'Please add a cover URL'],
+  },
+  UserId: {
+    type: mongoose.Schema.ObjectId,
+    ref: 'User',
+    required: [true],
```

Comment:
Note: when the **required** field is passed an array, it takes the form of [true, 'error message to show']